app/views/home/index.html.erb

```diff
@@ -8,7 +8,7 @@
 
 <h3>Add an idea</h3>
 
-<form>
+<form id='form'>
```

Comment:
This id is named very generally - you might to use a class instead or a more specific id name!